pkg/doc/did/doc_test.go

```diff
@@ -1293,6 +1275,125 @@ func TestDIDSchemas(t *testing.T) {
 	})
 }
 
+// nolint:lll
+func TestDoc_VerificationMethods(t *testing.T) {
+	didDocStr := `
+{
+  "@context": "https://w3id.org/did/v1",
+  "id": "did:elem:ropsten:EiAS3mqC4OLMKOwcz3ItIL7XfWduPT7q3Fa4vHgiCfSG2A",
+  "service": [
+    {
+      "id": "did:elem:ropsten:EiAS3mqC4OLMKOwcz3ItIL7XfWduPT7q3Fa4vHgiCfSG2A#openid",
+      "type": "OpenIdConnectVersion1.0Service",
+      "serviceEndpoint": "https://openid.example.com/"
+    }
+  ],
+  "assertionMethod": [
+    "did:elem:ropsten:EiAS3mqC4OLMKOwcz3ItIL7XfWduPT7q3Fa4vHgiCfSG2A#edv"
```

Comment:
Also need additional tests for relative IDs.

Reply:
OK, I will add them. This will need extra flag in `VerificationMethod` to allow equivalent conversion back to JSON from `did.Doc`.

Reply:
Not sure I fully understand the flag?

Reply:
I created a follow-up PR for relative IDs, they are not supported, and the test fails.
https://github.com/hyperledger/aries-framework-go/issues/1662